From 436ac9074def43ae09d7ecc28eec6cdc77a9d0e2 Mon Sep 17 00:00:00 2001
From: Chris Leech <cleech@redhat.com>
Date: Fri, 13 Sep 2013 16:56:51 -0700
Subject: [PATCH 65/65] fix hardened build of iscsiuio

The new iscsiuio code sets CFLAGS in configure.ac, wiping out the
environment setup by rpm.  Patch that out.

---
 Makefile              | 2 +-
 iscsiuio/configure    | 2 +-
 iscsiuio/configure.ac | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)
 mode change 100644 => 100755 iscsiuio/configure

diff --git a/iscsiuio/configure.ac b/iscsiuio/configure.ac
index e9a5e32..d9a6bdb 100644
--- a/iscsiuio/configure.ac
+++ b/iscsiuio/configure.ac
@@ -52,7 +52,6 @@ AC_LIBTOOL_DLOPEN
 # libtool stuff
 AC_PROG_LIBTOOL
 
-CFLAGS="-O2 -Wall"
 ## check for --enable-debug first before checking CFLAGS before
 ## so that we don't mix -O and -g
 AC_ARG_ENABLE(debug,
-- 
1.8.1.4

